packages/presentation/src/connect.ts

```diff
@@ -69,7 +76,22 @@ export async function connect (title: string): Promise<Client | undefined> {
   }
 
   // Update window title
-  document.title = [fetchMetadataLocalStorage(login.metadata.CurrentWorkspace), title].filter((it) => it).join(' - ')
+  function setDocumentTitle (title?: string): void {
```

Comment:
Le'ts move this code to Workbench plugin.

Reply:
Moved